src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-spring.jpa.hibernate.ddl-auto=none
+spring.jpa.hibernate.ddl-auto=update

```

Comment:
Why is this updated? We should have liquibase update the database schema, not hibernate.